src/QmlControls/ParameterEditorDialog.qml

```diff
@@ -138,6 +149,26 @@ QGCViewDialog {
                 }
             }
 
+            Column {
+                Repeater {
+                    id: bitmaskEditor
+                    model: fact.bitmaskStrings
+                    delegate : QGCCheckBox {
+                        text : modelData
+                        checked : fact.value & fact.bitmaskValues[index]
+                        function check_uncheck_all() {
+                            if (text.toLowerCase() == "all") {
```

Comment:
Nope. The value is "1", that's actually more generic.
Em 4 de abr de 2016 21:05, "Lucas De Marchi" <notifications@github.com>
escreveu:

> In src/QmlControls/ParameterEditorDialog.qml
> <https://github.com/mavlink/qgroundcontrol/pull/3119#discussion_r58469436>
> :
>
> > @@ -138,6 +149,26 @@ QGCViewDialog {
> >                  }
> >              }
> >
> > +            Column {
> > +                Repeater {
> > +                    id: bitmaskEditor
> > +                    model: fact.bitmaskStrings
> > +                    delegate : QGCCheckBox {
> > +                        text : modelData
> > +                        checked : fact.value & fact.bitmaskValues[index]
> > +                        function check_uncheck_all() {
> > +                            if (text.toLowerCase() == "all") {
>
> What I asked is that when you select "all" it actually selects all the
> values. I suggested a strcmp, but @DonLakeFlyer
> <https://github.com/DonLakeFlyer> is right that it's too generic. Maybe
> you can do the same but looking at the value instead?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mavlink/qgroundcontrol/pull/3119/files/0fd2dd0eaa41a4710ce34f981a38ba51a3e2c612#r58469436>
>